main/User_config.h

```diff
@@ -99,9 +99,11 @@ const byte mac[] = {0xDE, 0xED, 0xBA, 0xFE, 0x54, 0x95}; //W5100 ethernet shield
 #if defined(ESP8266) || defined(ESP32) || defined(__AVR_ATmega2560__) || defined(__AVR_ATmega1280__)
 #  define parameters_size     20
 #  define mqtt_topic_max_size 100
+#  define mqtt_max_packet_size 1024
 #else
 #  define parameters_size     15
 #  define mqtt_topic_max_size 50
+#  define mqtt_max_packet_size 128
 #endif
```

Comment:
You should apply clang format to the files, (we didn't updated the dev docs yet sorry).
Install the Clang-format extension to PIO and do a right click inside the main window of the file and click "format document"